result.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:bmi_calculator/constants.dart';
+import 'package:flutter/material.dart';
+import 'reusable_card.dart';
+import 'constants.dart';
+import 'bottom_button.dart';
+
+class ResultPage extends StatelessWidget {
+  const ResultPage(
+      {required this.resultText,
+      required this.resultVal,
+      required this.interp});
+
+  final String resultText;
+  final String resultVal;
+  final String interp;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Container(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Container(
+                width: double.infinity,
+                margin: EdgeInsets.fromLTRB(15, 50, 15, 0),
+                decoration: BoxDecoration(
+                  color: kColourUnselected,
+                ),
+                padding: EdgeInsets.fromLTRB(20, 40, 20, 20),
+                child: Text(
+                  "Your Result",
```

Comment:
Use const Text() instead of Text() to indicate that the widget will not be rebuilt during runtime, which can improve performance.